CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # dbt_fivetran_log v1.3.1
 
+## Feature Updates
+- This release introduces compatibility with **SQL Server** 🥳  🎆  🍾 ([PR #101](https://github.com/fivetran/dbt_fivetran_log/pull/101))
```

Comment:
should i get more specific with the changes i needed to add? like what i listed out in the PR description